### ISSUE TITLE
Pasting into teams light theme using dark theme version of Scrum Helper causes text to be unreadable (white)

### ISSUE THREAD

**Issue by GlennHS** [GlennHS/Scrum-Helper#4]
**Version:** Latest

**Affects Live?** Yes

**Description:** Pasting into teams light theme using dark theme version of Scrum Helper causes text to be unreadable (white)

**Comment by GlennHS** [comment]
Can probably fix for hotkey copying by simply changing the css style of the div to be color:black briefly, copying then changing back to white. Can do the same for clicking. Just wondering if having it be black will break _dark_ mode Teams next 😆 This one will probably take a little longer just because I'll need to test on my work laptop and test quite thoroughly

**Comment by esweby** [comment]
I think this would be a problem regardless as the people who use dark mode scrum helper use dark mode teams and so it displays correctly for them. Unless there's a way to strip styling from text upon copying?

**Comment by GlennHS** [comment]
Yeah, the issue with that is we lose other styling such as Bold, Italic etc. I would use a textarea as that has no styling. I did try formatting the message using Teams' markdown syntax a while ago but it just copies the mardown literally as opposed to parsing it
